common/app/services/Redirects.scala

```diff
@@ -0,0 +1,54 @@
+package services
+
+import com.gu.scanamo.{Scanamo, ScanamoAsync}
+import com.gu.scanamo.syntax._
+import common.{ExecutionContexts, Logging}
+import conf.Configuration
+
+import scala.concurrent.Future
+
+
+object Redirects {
```

Comment:
I think so yes, the only reason I didn't was consistency - we don't tend to use that naming convention (in `common` services anyway)